### PR TITLE
Bug 1163957 - [NFC]Can't share screenshot from notification preview page.,r=djf

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1163957

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8513a9c83f43b10231af522de419674fbf8ed4d0

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6b66fccc80b8c1bf6d5406a972496276ada38aee

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3acdb6176cf4933b7eaac4846c1de78d554f0e04

File: apps/system/js/app_window.js [hunk 1]
```diff
@@ -1278,7 +1278,7 @@
 
       this.screenshotOverlay.classList.remove('visible');
       this.screenshotOverlay.style.backgroundImage = '';
-      this.element.classList.remove('no-screenshot');
+      this.element && this.element.classList.remove('no-screenshot');
```
Comment (davidflanagan) [inline apps/system/js/app_window.js]: did you mean for this change to be part of this patch? Or is this an unrelated change?

It seems like a safe change, and I'm fine with it as long as you meant for it to be here.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c5723c02378ff3f60d7784fea7e2ee25592e3d09

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=faa6dbef38658a8f4e88c6a5e02a9f6179d9e823

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=00b2ca08b5c57c1eff2ef1fe8212d0329ca0704e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=80ce76a4f5ea9c54992356651f552518ca94d41e
